packages/microservices/server/server-kafka.ts

```diff
@@ -140,8 +140,8 @@ export class ServerKafka extends Server implements CustomTransportStrategy {
     replyPartition: string,
     correlationId: string,
   ): (data: any) => Promise<RecordMetadata[]> {
-    return (data: any) =>
-      this.sendMessage(data, replyTopic, replyPartition, correlationId);
+    return async (data: any) =>
+      await this.sendMessage(data, replyTopic, replyPartition, correlationId);
```

Comment:
```suggestion
     this.sendMessage(data, replyTopic, replyPartition, correlationId);
```

You don't need `await` here